### PR TITLE
[11.0][IMP] product_multi_price: add show multi-price access group

### PR DESCRIPTION
**Pull request by ernestotejeda**
Add a new access group to show/hide multi-prices list in product form view.

cc @Tecnativa TT24988

### CODE REVIEW

File: product_multi_price/hooks.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Copyright 2020 Tecnativa - Ernesto Tejeda
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, SUPERUSER_ID
+
+
+def post_init_hook(cr, registry):
+    env = api.Environment(cr, SUPERUSER_ID, {})
+    group_id = env.ref('product_multi_price.group_show_multi_prices').id
+    env['res.users'].search([]).write({'groups_id': [(4, group_id, None)]})
```
Comment (pedrobaeza) [inline product_multi_price/hooks.py]: Write also on inactive users, but only internal ones (`[("share", "=", False)]`).

File: product_multi_price/migrations/11.0.1.2.0/post-migration.py [hunk 2]
```diff
@@ -0,0 +1,9 @@
+#  Copyright 2020 Tecnativa - Ernesto Tejeda
```
Comment (pedrobaeza) [inline product_multi_price/migrations/11.0.1.2.0/post-migration.py]: You have to put a higher version than the existing one, and bump manually the module version, so there's no possibility of this not being triggered, even if ocabot is launched with other bump. Use `11.0.2.0.0`, as this is a major change in the module.

File: product_multi_price/migrations/11.0.1.2.0/post-migration.py [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+#  License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate(use_env=True)
+def migrate(env, version):
+    group_id = env.ref('product_multi_price.group_show_multi_prices').id
```
Comment (pedrobaeza) [inline product_multi_price/migrations/11.0.1.2.0/post-migration.py]: The idea here is not to duplicate code, but call the post_init_hook, and don't use openupgradelib, as it's not needed:
```python
from odoo.addons.product_multi_price.hooks import post_init_hook

def migrate(cr, version):
    post_init_hook(cr, False)
```

File: product_multi_price/views/product_views.xml [hunk 4]
```diff
@@ -10,6 +10,7 @@
                     widget="one2many_list"
                     context="{'default_product_id': product_variant_id,'tree_view_ref':'product_multi_price.product_multi_price_tree_view2', 'form_view_ref':'product_multi_price.product_multi_price_form_view'}"
                     attrs="{'invisible': [('product_variant_count', '&gt;', 1), ('is_product_variant', '=', False)]}"
+                    groups="product_multi_price.group_show_multi_prices"
```
Comment (pedrobaeza) [inline product_multi_price/views/product_views.xml]: Put the group in the view, not in the field.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

**Comment by ernestotejeda** [comment]
@pedrobaeza Changes done

**Review by pedrobaeza — APPROVED** [review]

**Review by rousseldenis — APPROVED** [review]
LGTM

**Comment by pedrobaeza** [comment]
Cherry-picked to v12 in 84166cdf1676e491397bdaeb91d64416e956fa66

**Pull request merged**